modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/DateRenderer.js

```diff
@@ -14,11 +14,18 @@
 
 import PropType from 'prop-types';
 
+import {getDateFromDateString, prettifyDateObject} from '../utils/dates';
+
 function DateRenderer({options, value}) {
 	if (!value) {
 		return null;
 	}
 
+	if (options.withoutTime) {
+		const dateObject = getDateFromDateString(value.split('T')[0]);
+		value = prettifyDateObject(dateObject);
```

Comment:
Thinking about it more, it's a pretty poor solution that we are assuming format of `value` to contain `T`. What if it's not in ISO format? Ideally `value` should be a `Date` object. But yeah, that goes beyond the scope of this task.

Reply:
Hey @markocikos, this is necessary because we don't want consider the time zone in `new Date(value)` at line 39.
In this case, we format the date to a format that doesn't consider the time zone: `new Date('yyyy/mm/dd);`.

Reply:
Oh I see, `toLocaleDateString` in `prettifyDateObject` makes the difference. 

:+1: 

[not a blocker for this PR] `prettifyDateObject`  is a pretty misleading name for that util :disappointed:

Reply:
That's true, I'm going to change this!

Reply:
In really, I reused this function, I noticed now. I prefer don't to change in this PR. That's ok for you?

Reply:
Yeah, that's fine :+1: